packages/build/src/plugins_core/functions/index.js

```diff
@@ -16,9 +16,12 @@ const coreCommand = async function ({
   constants: { FUNCTIONS_SRC: relativeFunctionsSrc, FUNCTIONS_DIST: relativeFunctionsDist },
   buildDir,
   logs,
+  childEnv,
+  netlifyConfig,
 }) {
   const functionsSrc = resolve(buildDir, relativeFunctionsSrc)
   const functionsDist = resolve(buildDir, relativeFunctionsDist)
+  const { externalModules } = netlifyConfig.build
```

Comment:
Perhaps this should be something like `experimentalExternalModules` until we settle on a definite plan for allowing users to configure esbuild?

Reply:
Yes, that'd be a good idea :+1: 

After we settle on something definitive, if we do use a `netlify.toml` configuration property, we should also [validate it](https://github.com/netlify/build/blob/master/packages/config/src/validate/validations.js) in `@netlify/config`

Reply:
Absolutely!